app/domain/person/block_service.rb

```diff
@@ -0,0 +1,27 @@
+class Person::BlockService
```

Comment:
@carlobeltrame : Ich habe mich hier auch wieder dazu entschieden, die Funktionalität in einen Service zu extrahieren, weil z.B. das blockieren der Personen sowohl aus dem UI als auch aus dem Job kommen könnte und es so einfacher zu testen ist. Für bessere Namensvorschläge bin ich offen.